src/libraries/Microsoft.Extensions.Options/tests/SourceGeneration.Unit.Tests/OptionsRuntimeTests.cs

```diff
@@ -266,7 +266,7 @@ public void TestNewDataAnnotationFailures()
             }, result.Failures);
         }
 #endif // NET8_0_OR_GREATER
-
+        [SkipOnPlatform(TestPlatforms.LinuxBionic, "https://github.com/dotnet/runtime/issues/93723")]
```

Comment:
For whatever reason, this doesn't seem to be respected on the latest CI run

Reply:
Correct, here is the latest run: https://dev.azure.com/dnceng-public/public/_build/results?buildId=444522&view=logs&j=8ce12db4-5178-5d5b-6012-cb5f9bb20b0f.

Closing this PR. If the same error is encountered we will reopen it.